mantidproject/mantid
# Vesuvio  - Range of Run Numbers as Input

LoadVesuvio and VesuvioDiffractionReduction have been modified to allow a range of run numbers to be input for `FoilOut`, `FoilIn` and `FoilInOut` modes. The runs are summed and result normalised. 

**To test:**

The InstrumentParFile can be found at: 
`\\olympic\babylon5\Public\Louise McCann\IP_2015_06_Chimney_header.par`

Ensure that the Data Archive Service can be accessed.
Run LoadVesuvio with the following parameters:

- Filename =  23893-23912
- SpectrumList = 3-134
- Mode =  `FoilOut`, `FoilIn` and `FoilInOut`
- InstrumentParFile = Previously mentioned file available from babylon5 - If you can't find the file in file browser, ensure the file browser is searching for .par files. It defaults to looking for .dat files.
- OutputWorkspace = outWs

Fixes #17891.

[Release Notes](https://github.com/mantidproject/mantid/blob/079e7ca5536cdb31087048e9b62a204eceae2587/docs/source/release/v3.9.0/indirect_inelastic.rst)

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
